composer.json

```diff
@@ -38,16 +38,23 @@
         }
     ],
     "require": {
-        "php": "~5.5 || ~7.0",
-        "doctrine/doctrine-module": "~1.0",
-        "doctrine/mongodb-odm": "~1.0",
-        "zendframework/zend-mvc": "2.*",
-        "zendframework/zend-servicemanager": "2.*",
-        "zendframework/zend-stdlib": "2.*"
+        "php":                               "~5.6 || ~7.0",
+        "doctrine/doctrine-module":          "dev-master",
```

Comment:
No :( not yet, @gianarb merged `DoctrineModule` and `DoctrineORMModule` without tagging.
Please see his comment here: https://github.com/doctrine/DoctrineORMModule/issues/491#issuecomment-236649515